File: app/src/main/java/org/mhsgv/portal/form/FormFragment.java
package org.mhsgv.portal.form;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

public class FormFragment extends Fragment {

    protected View createView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState, int layout) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(layout, container, false);
        return view;
    }

}

File: app/src/main/java/org/mhsgv/portal/form/FormFragment.kt
package org.mhsgv.portal.form

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

open class FormFragment : Fragment() {
    protected fun createView(inflater: LayoutInflater, container: ViewGroup?,
                             savedInstanceState: Bundle?, layout: Int): View {
        // Inflate the layout for this fragment
        return inflater.inflate(layout, container, false)
    }
}